io.spectra.sudoku_solver.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id" : "io.tachyon.sudoku_solver",
```

Comment:
`tachyon.io` doesn't seem to be related to this either.

I'll rephrase. One doesn't simply pick a random domain.

The documentation state that the domain should be under the control of the upstream developers and even offer choices for using code hosting provided domain for the app-id.

Reply:
I've updated the the appid again, this time i'm using code hosting domain appid so it'll hopefully be fine.

Reply:
you didn't.

Reply:
> you didn't.

i didnt push the changes here yet, i want to resolve https://github.com/flathub/flathub/pull/3805#discussion_r1070307912 before pushing anything here.

Reply:
The new app id is gonna be `com.gitlab.cyberphantom52.sudoku_solver`.

Reply:
it should be `io.gitlab.cyberphantom52.sudoku_solver` as per the documentation

(the case of gitlab is identical to github)

https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
https://github.com/cyberphantom52/flathub/commit/57c5698bdc252f6c193900d7533c3427db836948

Should address this